pkg/cli/login/loginoptions.go

```diff
@@ -262,6 +280,24 @@ func (o *LoginOptions) gatherAuthInfo() error {
 		}
 	}
 
+	if o.OIDCExecPluginType == string(OC) {
```

Comment:
no action required:
in the future, design this so that you just pick a provider config function from a map